packages/database/src/core/view/QueryParams.ts

```diff
@@ -191,10 +194,12 @@ export class QueryParams {
     copy.limitSet_ = this.limitSet_;
     copy.limit_ = this.limit_;
     copy.startSet_ = this.startSet_;
```

Comment:
Let's remove `hasStartAfter()` and `hasEndBefore()`, they aren't used.

Reply:
Done. I'll do a follow up PR to remove `NextPushId.successor` et al.